### PR TITLE
Compressed air centrifugal

### PR DESCRIPTION
**Pull request by omerbaa**
Pull request overview
---------------------

### CODE REVIEW

File: binding.gyp [hunk 1]
```diff
@@ -298,6 +298,34 @@
                     'cflags_cc': ['-fexceptions']
                 }]
             ]
+        },
+        {
+          "target_name": "caCentrifugal",
```
Comment (rmroot) [inline binding.gyp:303]: This looks like you are going to make a different binding target for each compressed air algorithm. It'd be prefered and probably easier if we held all the compressed algorithm bindings under a single target (something like "compressedAir").

File: main.cpp [hunk 2]
```diff
@@ -1,8 +1,70 @@
-#include <iostream>
+/*#include <iostream>
 
 int main() {
     std::cout << "Run 'doxygen Doxyfile.in' from the root project directory to generate documentation." << std::endl;
 	std::cout << "View the documentation by opening docs/html/index.html in a web browser." << std::endl;
 	return 0;
+}*/
+
+#include <iostream>
```
Comment (rmroot) [inline main.cpp:9]: I think this code was used for testing but we don't want to modify the main.cpp file

**Review by rmroot — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (omerbaa) [reply main.cpp]: the test code should not have been checked in

File: binding.gyp (hunk 1 above)
Comment (omerbaa) [reply binding.gyp]: was not sure on how different the compressors are, 
started with different binding for each,
modified to keep single binding for all compressors

**Review by rmroot — APPROVED** [review]

**Pull request merged**